bootstrap/farmos.py

```
@@ -0,0 +1,174 @@
+def is_farmos_ready(db_conf: dict[str, str]) -> bool:
+    """운영에 필요한 최소 상태를 확인한다.
+
+    기준:
+    - 필수 테이블 존재
+    - users 테이블에 테스트 계정 2명 이상 존재
+    """
+    for table in FARMOS_TABLES:
+        if not table_exists(db_conf, table):
+            return False
+    for table, expected in EXPECTED_ROW_COUNTS.items():
+        actual = int(psql_query(db_conf, f"SELECT COUNT(*) FROM {table};") or "0")
```

Ruff (0.15.10)

[error] Possible SQL injection vector through string-based query construction

(S608)